dev-libs/optix/optix-7.4.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV=${PV:0:3}
+
+DESCRIPTION="NVIDIA Ray Tracing Engine"
+HOMEPAGE="https://developer.nvidia.com/optix"
+SRC_URI="NVIDIA-OptiX-SDK-${PV}-linux64-x86_64.sh"
+S="${WORKDIR}"
+
+HEADER_INTERNAL_FILES="
+optix_7_device_impl.h
+optix_7_device_impl_exception.h
+optix_7_device_impl_transformations.h
+"
+
+HEADER_FILES="
+optix.h
+optix_7_device.h
+optix_7_host.h
+optix_7_types.h
+optix_denoiser_tiling.h
+optix_device.h
+optix_function_table.h
+optix_function_table_definition.h
+optix_host.h
+optix_stack_size.h
+optix_stubs.h
+optix_types.h
+"
+
+for i in ${HEADER_INTERNAL_FILES}; do
+	SRC_URI="${SRC_URI}
+		header-only? ( https://developer.download.nvidia.com/redist/optix/v${MY_PV}/internal/${i} -> ${P}-internal-${i} )"
```

Comment:
Was thinking about changing this to use `+=`, but wasn't sure if SRC_URI can use it. Happy to change this.

Reply:
Yeah it's fine, tools will use the metadata cache processed by bash rather than try to parse this, so how it was constructed doesn't matter beside style.